mono/tests/typeloadexception.cs

```diff
@@ -0,0 +1,46 @@
+using System;
```

Comment:
The file name is too generic. Also, there is already a test in the coreclr repo for this.

Reply:
Removed the test